src/components/RulesetFormSections/LabelFieldArray/LabelFieldArray.js

```diff
@@ -52,6 +77,23 @@ const LabelFieldArray = () => {
     return dataOptions.filter(({ label }) => label.search(regex) !== -1);
   };
 
+  const dateTokens = () => {
+    return [
+      '{{chronology1.weekday}}',
+      '{{chronology1.monthDay}}',
+      '{{chronology1.month}}',
+      '{{chronology1.year}}',
+    ].join(' ');
+  };
+
+  const monthTokens = () => {
+    return ['{{chronology1.month}}', '{{chronology1.year}}'].join(' ');
+  };
+
+  const yearTokens = () => {
+    return ['{{chronology1.year}}'];
+  };
```

Comment:
Still need to have these moved to the `ChronologyField` component, these don't need to be passed down as props